diff --git a/packages/stk/stk_util/stk_util/util/ReportHandler.cpp b/packages/stk/stk_util/stk_util/util/ReportHandler.cpp
index c83e423b..09c5dad7 100644
--- a/packages/stk/stk_util/stk_util/util/ReportHandler.cpp
+++ b/packages/stk/stk_util/stk_util/util/ReportHandler.cpp
@@ -43,7 +43,7 @@
 #include "boost/version.hpp"
 #if BOOST_VERSION >= 106500
 #ifndef STK_NO_BOOST_STACKTRACE
-#define STK_HAVE_BOOST_STACKTRACE
+//#define STK_HAVE_BOOST_STACKTRACE
 #endif
 #endif
 #endif
